person.rb

```diff
@@ -1,5 +1,11 @@
-# class representing a person
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
 class Person
+  include Nameable
+
   attr_accessor :name, :age
   attr_reader :id
 
```

Comment:
[OPTIONAL]
Another optional comment that I have for you is that the Person class should have its own correct_name method, because you have not implemented that requirement. The correct_name method should return the name of the person without any modification.

To implement this change, you need to add a correct_name method to the Person class, like this:

```ruby
class Person < Nameable
  .
  .

    def correct_name
       name
    end
  .
  .
end
```

This is also an optional change request, but I think it would make your code more clear and complete. Please let me know if you have any questions or feedback about this suggestion. I look forward to seeing your next submission. 😊